packages/app/navigation/native/tab.tsx

```diff
@@ -21,6 +24,21 @@ export function DashboardNavigation() {
       <Tab.Screen component={SearchScreen} name="search" />
       <Tab.Screen component={ChartScreen} name="chart" />
       <Tab.Screen component={ProfileNavigation} name="profile" />
+      <Tab.Screen
+        options={{
+          headerShown: true,
+          title: "Beatmaker",
+          headerStyle: {
+            backgroundColor: tw.color("blue-dark"),
+          },
+          headerTintColor: tw.color("white"),
+          headerTitleStyle: {
+            fontWeight: "bold",
+          },
+        }}
+        component={BeatmakerScreen}
+        name="beatmaker"
+      />
```

Comment:
it definitely shouldn't be a tab screen. Tab screens are screens that we can switch between using the bottom navigation bar. Beatmaker should be a part of the stack navigator.

Reply:
Can't find a better place to put it

Reply:
added a new stack navigation